3p/ampcontext.js

```diff
@@ -100,7 +100,7 @@ export class AbstractAmpContext {
     /** @protected {!IframeMessagingClient} */
     this.client_ = new IframeMessagingClient(win);
     this.client_.setHostWindow(this.getHostWindow_());
-    this.client_.setSentinel(dev().assertString(this.sentinel));
```

Comment:
I was basing this on feedback from @keithwrightbos at https://github.com/ampproject/amphtml/pull/10596#discussion_r130183444
Keith, do you want to defer to Justin, or...?